synapse/models/orgs.py

```diff
@@ -70,7 +66,7 @@ def getModelDefs(self):
                 }),
                 ('ou:industry:type:taxonomy', ('taxonomy', {}), {
```

Comment:
This needs to be a form (existing story to add in 2.x, this is blocking ou:industry creation for analysts :-) ).

Reply:
How is this lacking a form a blocker for industry creation?

Reply:
Added SYN-7957 to address 2.x. Looks like it is ou:industry:type:taxonomy is still only a type in 3.x / needs to be added as a form?